proposals/gc/MVP.md

```diff
@@ -529,10 +567,22 @@ Tentatively, support a type of guaranteed unboxed scalars.
   - traps if the operand is null
 
 
-#### Classification
+#### External conversion
+
+* `extern.internalize` converts an external value into the internal representation
```

Comment:
In the discussions, in was mentioned that it would be useful for performance to include direct transformations to more specific abstract types. This might be best as a post-MVP feature though.